refactor: 로그인 화면 Figma 디자인에 맞게 수정

LeeKeonSoo opened the pull request:
## 🧑‍🚀 PR 요약
#### 작업 내용
해당 pr에서 작업한 내역을 적어주세요.

- LoginButton 이미지도 넣을 수 있게 구조 변경.
- wegg_icon, wegg_text, google_icon, kakao_icon 추가.

#### 📌 변경 사항
변경사항 및 주의 사항 (모듈 설치 등)을 적어주세요.

- wegg_icon 파일 변경. 이름은 동일함.

#### 📸 ScreenShot
- [x] iPhone 16 Pro와 iPhone 13mini에서 테스트 완료
- 스크린샷은 하나의 기기에서 진행한 내용만 첨부(ex. iPhone 16Pro)

![Simulator Screenshot - iPhone 16 Pro - 2025-01-25 at 19 50 55](https://github.com/user-attachments/assets/814ba77b-7e1f-46f2-8b86-bb90f7f0360e)


#### 🙏 리뷰 요청 사항
- 없습니다.

#### 🔗 Linked Issue
close #38 

#### ✅ PR check list
```
- commit message가 적절한지 확인해주세요. 
- 마지막으로 Coding Convention을 준수했는지 확인해주세요.
- 적절한 branch로 요청했는지 확인해주세요.
- Assignees, Label을 붙여주세요.
- 가능한 이슈를 Link 해주세요.
- PR이 승인된 경우 해당 브랜치는 삭제해주세요.
```

Commit:
feat: Asset에 Naver, Kakao 로그인 이미지 추가, LoginView 기본 구성

Commit:
레이아웃 완성

Commit:
아이콘 사이즈 조정

Commit:
레이아웃 맞춤

Commit:
feat: 로그인 화면 구현

Commit:
Merge branch 'main' of https://github.com/UMCWegg/iOS_Wegg into addfont

Commit:
add font

Commit:
feat: 로그인 화면 구현

Commit:
feat: 로그인 화면 구현

Commit:
레이아웃 정리

Commit:
feat: 로그인 화면 구현

Commit:
네이버 한국어로 변경

Commit:
[feat] blue, yellow 색상 추가

Commit:
Feat: 다른 방법으로 로그인 화면 구현

Commit:
[bug] 로그인 버튼 글자 색상 정상화.

Commit:
fix: 색상, 폰트 main 브랜치에서 병합 충돌 해결

Commit:
refactor: LoginButton. Component 하위 폴더로 구조 변경

Commit:
style: MARK 순서 변경

Commit:
refactor: Navigation 관련 로직 제거

Commit:
Style: 내가 만든 blue, yellow 색상 제거

Commit:
Fix: TextField 배경화면 안보이는 문제 해결

Commit:
Style: 버튼 색 이름 Camel Case로 변경

Commit:
refactor: 로그인에 사용하는 TextField Components 하위로 모듈화

Commit:
refactor: EmailLoginView 안에 TextField. 커스텀 한 LoginTextField로 변경

Commit:
feat: FindPasswordView, ViewController 파일 추가

Commit:
feat: 비밀번호 찾기 구현 중간 저장.

Commit:
refactor: TextField heightAnchor 추가

Commit:
feat: 비밀번호 찾기 화면 레이아웃 구성 완료

Commit:
style: 로그인 버튼 예외시 색상 systemBlue로 변경

Commit:
refactor: 인증번호 보내기, 재발송 버튼 AutoLayout 조정

Commit:
style: SignUp 폴더 Screen 하위 폴더로 구성.

Commit:
feat: View, ViewController 파일 추가.

Commit:
style: 파일명 오타 수정

Commit:
feat: CheckButton 모듈화해서 구현

Commit:
feat: ServiceAgreementView 중간 저장. API 구현으로 우선순위 변경.

Commit:
feat: Login 관련 Model 추가

Commit:
refactor: Merge branch 'main' of https://github.com/UMCWegg/iOS_Wegg into mergetest

# Conflicts:
#	iOS_Wegg/iOS_Wegg.xcodeproj/project.pbxproj
#	iOS_Wegg/iOS_Wegg/Component/TextField/Login/LoginTextField.swift
#	iOS_Wegg/iOS_Wegg/Screen/Login/View/EmailLoginView.swift

Commit:
fix: Merge 과정 완료

Commit:
Merge branch 'mergetest' into feature/serviceAgreement

Commit:
fix: Merge 잔재 제거

Commit:
docs: README에 네이버 로그인 SDK 추가.

Commit:
feat: Login과 User 정보에 대한 Model 추가.

Commit:
refactor: LoginRequest, Response로 파일명 변경

Commit:
feat: LoginManager로 파일 분리 이후 KakaoLoginManager 작성.

Commit:
style: 이전 Commit Kakao가 아닌 Naver Login Manager

Commit:
feat: README에 카카오 로그인 SDK 문서 추가. 각 LoginManager 생성

Commit:
feat: 소셜 로그인, 이메일 로그인 1차 구현 완료. 실행 이전.

Commit:
feat: AuthService에 Parameter 양식에 맞게 로그인 성공 실패에 따른 print문 작성

Commit:
fix: 각 LoginManager 오류 제거

Commit:
feat: AppDelegate에 KakaoSDK 앱 키 란 추가

Commit:
feat: NaverLoginManager 제거. GoogleLoginManager 추가.

Commit:
feat: GoogleLoginManager로 작동할 수 있도록 기존 NaverLoginManager 코드 수정 및 변수명 변경

Commit:
style: SwiftLint 규격 맞춤

Commit:
feat: 구글 로그인 ClientID 추가.

Commit:
docs: 기술 스택 추가

Commit:
feat: 구글 문서 보고 구글 로그인 구현

Commit:
docs: Info.plist에 URL Scheme 추가

Commit:
feat: UserDefaults에 계정 저장 로직 추가

Commit:
refactor: Login Button, TextField 코드 개선

Commit:
refactor: 디자인 바뀐 부분 코드 제거

Commit:
feat: Figma에 맞는 이메일 로그인 화면 구성

Commit:
feat: EmailLoginButton에 로그인 로직 연결

Commit:
fix: EmailLoginVC에 누락된 함수 추가

Commit:
fix: Merge branch 'main' of https://github.com/UMCWegg/iOS_Wegg into feature/LoginAPI

# Conflicts:
#	iOS_Wegg/iOS_Wegg.xcodeproj/project.pbxproj
#	iOS_Wegg/iOS_Wegg/App/SceneDelegate.swift
#	iOS_Wegg/iOS_Wegg/Resources/Assets.xcassets/Images/Login/BackButton.imageset/Contents.json
#	iOS_Wegg/iOS_Wegg/Resources/Assets.xcassets/Images/Login/wegg_icon.imageset/Contents.json
#	iOS_Wegg/iOS_Wegg/Resources/Assets.xcassets/Images/wegg_icon.imageset/Contents.json
#	iOS_Wegg/iOS_Wegg/Resources/Assets.xcassets/Images/wegg_icon.imageset/wegg_icon.png
#	iOS_Wegg/iOS_Wegg/Resources/Info.plist

Commit:
feat: 로그인시 Token 저장 로직 추가

Commit:
fix: extensions Compile 과정에 들어가지 않아 작동하지 않는 이슈 해결. 동시에 불필요한 색상 제거

Commit:
refactor: FindPasswordView에서 PasswordRecoveryEmailView로 이름 변경

Commit:
refactor: 비밀번호 찾기 이메일, 인증번호 입력 화면 구현

Commit:
style: 파일명 수정

Commit:
feat: LoginButton Image 넣을 수 있도록 로직 추가

Commit:
style: Login에 사용되는 asset들 추가

Commit:
refactor: Image의 유무에 따라 LoginButton의 구조 다르게 리팩토링.

Commit:
fix: 로그인 버튼 아이콘이랑 텍스트 겹치는 버그 해결

Commit:
fix: Xcode에서 파일 보이지 않는 이슈 해결.

jwon0523 reviewed (commented):
- AppDelegate에 민감한 정보(App Key나 클라이언트 아이디) xcconfig에 작성해서 민감한 정보는 숨겨주세요

Commit:
fix: 메인 브랜치와 충돌 해결
Merge commit '7ce1ebd0878222d8c60e61d9707e570e06093161' into refactor/LoginView

# Conflicts:
#	README.md
#	iOS_Wegg/iOS_Wegg.xcodeproj/project.pbxproj
#	iOS_Wegg/iOS_Wegg/App/SceneDelegate.swift
#	iOS_Wegg/iOS_Wegg/Component/Button/Login/LoginButton.swift
#	iOS_Wegg/iOS_Wegg/Resources/Assets.xcassets/Colors/Primary.colorset/Contents.json
#	iOS_Wegg/iOS_Wegg/Resources/Assets.xcassets/Images/wegg_icon.imageset/wegg_icon.png
#	iOS_Wegg/iOS_Wegg/Screen/Login/View/EmailLoginView.swift
#	iOS_Wegg/iOS_Wegg/Screen/Login/View/FindPasswordView.swift
#	iOS_Wegg/iOS_Wegg/Screen/Login/View/LoginView.swift
#	iOS_Wegg/iOS_Wegg/Screen/Login/View/PasswordResetEmailView.swift
#	iOS_Wegg/iOS_Wegg/Screen/Login/View/PasswordResetVerificationView.swift
#	iOS_Wegg/iOS_Wegg/Screen/Login/ViewController/LoginViewController.swift
